addons/source-python/packages/source-python/weapons/entity.py

```diff
@@ -5,10 +5,14 @@
 # =============================================================================
 # >> IMPORTS
 # =============================================================================
-from importlib import import_module
-from core import GAME_NAME
-from core import SOURCE_ENGINE
-from paths import SP_PACKAGES_PATH
+# Source.Python Imports
+#   Core
+from core import engine_import
+#   Entities
+from entities.entity import Entity
+#   Weapons
+from _weapons._entity import WeaponMixin
```

Comment:
When building the documentation, the _weapons module gives an ModuleNotFoundError:

```python
E:\Servers\cstrike\cstrike\addons\source-python\docs\source-python\source\developing\modules\weapons.entity.rst:9: WARNING: autodoc: failed to import class 'WeaponMixin' from module '_weapons._entity'; the following exception was raised:
Traceback (most recent call last):
  File "E:\Servers\cstrike\cstrike\addons\source-python\packages\site-packages\sphinx\ext\autodoc.py", line 518, in import_object
    __import__(self.modname)
ModuleNotFoundError: No module named '_weapons'
```

Reply:
Seems like you are running an old build that date before the commit that introduced the _weapons._entity module: https://github.com/Source-Python-Dev-Team/Source.Python/commit/d9b7b6b7cc5431650a7db1b2a47f0d916f04973d
If I correct that `__getattribute__` to `__getattr__` that was pointed above, the doc is building fine for me on TF2.